tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -266,6 +281,22 @@ def _test_storage_type(storage_type, fsx_fs_id, region):
         assert_that(get_storage_type(fsx_fs_id, region)).is_equal_to("SSD")
 
 
+def get_deployment_type(fsx_fs_id, region):
```

Comment:
internal convention: when a function is "private" for the module we're adding an underscore as prefix.

Reply:
Done